specification/request-specification.md

```diff
@@ -29,6 +29,8 @@ Below are the required parameters that will be encrypted using AES256 encryption
 | redirect           | String             | Requestor Url that AA needs to call back after the user has provided consent in the AA domain. The value of this parameter should be URL encoded if the value contains url parameters. This is required in order to remove ambiguity between the parameters of ecreq (separated by ‘&’ character) with the parameters in the redirect url. |
 | srcref             | Array              | Array of consent handle id(s), as returned by AA server to the /Consent request api invoked by the FIU(s) on the AA prior to this redirection call.                                                                                                                                                                                        |
 | fipid              | Array              | Optional, can be blank. If FIU already has the information that citizen wants to discover accounts of one or more FIP(s) beforehand, then they can pass this info to AA. It saves user clicks and enhances journey experience                                                                                                              |
+| emailid            | String              | Optional, can be blank. If FIU wants to notifiy the consent creation and subsequent notifications to the citizen thorugh email
+| PAN              | String              | Optional, can be blank. PAN of the citizen for discovery of certain FI categories like Mutual Funds
```

Comment:
In order to keep the field naming consistent, let's use lowercase for this field name as well.